ci/Jenkinsfile.linux

```diff
@@ -120,7 +128,7 @@ pipeline {
     }
 
     stage('E2E') {
-      when { expression { utils.isPRBuild() } }
+      when { expression { utils.isPRBuild() && !params.NWAKU } }
```

Comment:
hmm this did not work as expected.
The nwaku linux package job did trigger E2E and it should not have.

<img width="969" height="402" alt="Screenshot 2025-08-11 at 12 39 07 PM" src="https://github.com/user-attachments/assets/2c6e7d55-51b4-4f31-af8b-8c301ac978e9" />

Reply:
ah my bad this should have been `!params.USE_NWAKU ` and not `!params.NWAKU `